planner/core/find_best_task.go

```diff
@@ -1025,7 +1025,7 @@ func (ds *DataSource) getOriginalPhysicalTableScan(prop *property.PhysicalProper
 		ts.StoreType = kv.TiFlash
 		ts.filterCondition = append(ts.filterCondition, ts.AccessCondition...)
 		ts.AccessCondition = nil
-		ts.Ranges = ranger.FullNotNullRange()
+		ts.Ranges = ranger.FullIntRange(false)
```

Comment:
Will this be influenced?
If it's primary key as handle. It would be not null.
If it's hidden handle, it would not null too.